src/Sinch/Verification/SinchVerification.cs

```diff
@@ -158,18 +158,23 @@ public async Task<StartSmsVerificationResponse> StartSms(StartSmsVerificationReq
                 };
             }
 
+            SmsOptions? options = null;
+            if (request.CodeType != null || request.Expiry != null || request.Template != null)
```

Comment:
This looks better indeed. Can you add a test related to this code update (which is BTW totally unrelated to the PR title and should have gone in a dedicated one) ?

Reply:
Added new mock for the case: https://github.com/sinch/doppelganger/pull/34

Reply:
> which is BTW totally unrelated to the PR title and should have gone in a dedicated one
The change is seemingly small improvement without affecting functionality, so it seemed fine to add it here in this pretty small PR

Reply:
Your previous comment highlights perfectly what I was saying: we are now in a situation where an update for the **Conversation** API is blocked by an external pull request for a mock related to the **Verification** API.